libs/domain/picking/src/components/picking/picking.component.ts

```diff
@@ -37,6 +42,9 @@ export class PickingComponent extends PickingListMixin(LitElement) {
     })
   );
 
+  protected productCardRef: Ref<PickingProductCardComponent> = createRef();
+  protected tabRef: Ref<TabComponent> = createRef();
```

Comment:
Let's rename it to reflect that it's specific tab:
```suggestion
  protected notPickedTabRef: Ref<TabComponent> = createRef();
```